m4/pbs_decl_ppoll.m4

```diff
@@ -56,13 +57,13 @@ int main()
 {
 	sigset_t allsigs;       
 	int n;
-	int fd[2];
+	[int fd[2];]
```

Comment:
What do the brackets represent?

Reply:
Apparently, this is the kind of "escape"ing required to have the generated C code actually have the [] (brackets) for the fd array. Without that, the code generated is fd2; and thus compilation fails, resulting in configure deciding that ppoll is not present in the system.

Reply:
I missed the fact it was part of an m4 macro the first time I looked. The use of brackets indicates a text block that is not to be evaluated. Please enclose the entire C source code block with brackets as opposed to individual lines. That should improve readability.

Reply:
Thanks for explaining. I will put the whole code in brackets